src/script/hydra-create-user

```diff
@@ -25,27 +25,37 @@ exists, roles are added to the existing roles unless --wipe-roles is
 specified.  If --rename-from is given, the specified account is
 renamed.
 
-* PASSWORD HASH
-The password hash should be an Argon2id hash, which can be generated
-via:
+* Specifying Passwords
+
+** Specifying a Hash
+
+You can generate a password hash and provide the hash as well. This
+is useful so a user can send the administrator their password pre-hashed,
+allowing the user to get their preferred password without exposing it
+to the administrator.
+
+Hydra uses Argon2id hashes, which can be generated like so:
 
     $ nix-shell -p libargon2
-    [nix-shell]$ argon2 "$(LC_ALL=C tr -dc '[:alnum:]' < /dev/urandom | head -c16)" -id -t 3 -k 262144 -p 1 -l 16 -e
+    [nix-shell]$ tr -d \\\\n | argon2 "$(LC_ALL=C tr -dc '[:alnum:]' < /dev/urandom | head -c16)" -id -t 3 -k 262144 -p 1 -l 16 -e
     foobar
     Ctrl^D
     $argon2id$v=19$m=262144,t=3,p=1$NFU1QXJRNnc4V1BhQ0NJQg$6GHqjqv5cNDDwZqrqUD0zQ
 
-SHA1 is also accepted, but SHA1 support is deprecated and the user's
-password will be upgraded to Argon2id on first login.
+Example:
 
+  $ hydra-create-user alice --password-hash '$argon2id$v=19$m=262144,t=3,p=1$NFU1QXJRNnc4V1BhQ0NJQg$6GHqjqv5cNDDwZqrqUD0zQ' --role admin
 
-Examples:
+SHA1 is also accepted, but SHA1 support is deprecated and the user's
+password will be upgraded to Argon2id on first login.
 
-Create a user with an argon2 password:
+** Specifying a plain-text password as an argument (dangerous)
 
-  $ hydra-create-user alice --password-hash '$argon2id$v=19$m=262144,t=3,p=1$NFU1QXJRNnc4V1BhQ0NJQg$6GHqjqv5cNDDwZqrqUD0zQ' --role admin
+This option is dangerous and should not be used: it exposes passwords to
+other users on the system. This option only exists for backwards
```

Comment:
I wonder, should we worry about backwards compatibility with a dangerous option like this? Would it not be better and safer to just remove this altogether? Or maybe (somehow) only make it usable in a dev server...? (If that's even possible to detect)

Reply:
I don't want to drop it right away, but I did add an admonition:

```
[nix-shell:~/projects/github.com/NixOS/hydra]$ hydra-create-user foobar --password oeusnth
creating new user `foobar'
Submitting plaintext passwords as arguments is deprecated and will be removed. See --help for alternatives.
```

Reply:
Should we also change the invocation in `foreman/start-hydra.sh` to not use the raw `--password` (so we don't see that when running hydra in a dev shell)?